tools/clone_nonbillables_and_process.sh

```diff
@@ -28,9 +28,19 @@ if [ ! -d ./non-billable-projects ]; then
 fi
 
 INVOICE_MONTH=$(date --date="$(date +%Y-%m-01) -1 month" +%Y-%m)
+export S3_ENDPOINT
+export S3_KEY_ID
+export S3_APP_KEY
+export S3_BUCKET_NAME
 python process_report/process_report.py \
+    --fetch-from-s3 \
+    --upload-to-s3 \
     --invoice-month $INVOICE_MONTH \
     --pi-file ./non-billable-projects/pi.txt \
     --projects-file ./non-billable-projects/projects.txt \
     --timed-projects-file ./non-billable-projects/timed_projects.txt \
-    --old-pi-file old_pi.csv
+    --nonbillable-file "NERC (Nonbillable)" \
+    --output-file "NERC" \
+    --output-folder "PI Invoices" \
+    --BU-invoice-file "NERC BU" \
```

Comment:
You are missing the file extensions in all arguments that are a file here.

Reply:
Since all the files in the container will be ephemeral, why do we need to include extensions?

Reply:
You are assuming that this script will only ever be used inside the container.